config/v1.5/aws-k8s-cni.yaml

```diff
@@ -101,7 +101,7 @@ spec:
             - name: AWS_VPC_K8S_CNI_VETHPREFIX
               value: eni
             - name: AWS_VPC_K8S_CNI_MTU
-              value: 9001
+              value: "9001"
```

Comment:
interesting. this was a problem?

Reply:
Yeah, for some reason I got an error applying the config without the quotes.